core/core-dotspacemacs.el

```diff
@@ -162,6 +162,9 @@ size to make separators look not too crappy.")
 (defvar dotspacemacs-ex-substitute-global nil
   "If non nil, inverse the meaning of `g' in `:substitute' Evil ex-command.")
 
+(defvar dotspacemacs-override-evil-folding nil
```

Comment:
That sounds nice. Do you want to have a transient state for `'evil` folding method then?